terraform/cloudsql/mysql/bind/data.tf

```diff
@@ -0,0 +1,13 @@
+resource "random_string" "username" {
```

Comment:
nitpick: resources belong in the main.tf; data is for local variables and calculations